packages/react-day-picker/src/types/DayPickerProps.ts

```diff
@@ -1,15 +1,240 @@
+   * Paginate the month navigation displaying the [[numberOfMonths]] at time.
+   */
+  pagedNavigation?: boolean;
+  /**
+   * Render the months in reversed order (when [[numberOfMonths]] is greater
+   * than `1`) to display the most recent month first.
+   */
+  reverseMonths?: boolean;
+  // #endregion
+
+  // #region customization props
+  /**
+   * Change the layout of the caption:
+   *
+   * - `buttons` (default): display prev/right buttons
+   * - `dropdown`: display drop-downs to change the month and the year
+   *
+   * **Note** `dropdown` is valid only when `fromDate/fromMonth/fromYear` and
+   * `toDate/toMonth/toYear` are set.
+   */
+  captionLayout?: CaptionLayout;
+  /**
+   * Display six weeks per months, regardless the month’s number of weeks.
+   * To use this prop, [[showOutsideDays]] must be set. Default to `false`.
+   */
+  fixedWeeks?: boolean;
+  /**
+   * Hide the month’s head displaying the weekday names.
+   */
+  hideHead?: boolean;
+  /**
+   * Show the outside days.  An outside day is a day falling in the next or the
+   * previous month. Default is `false`.
+   */
+  showOutsideDays?: boolean;
+  /**
+   * Show the week numbers column. Default to `false`.
+   */
+  showWeekNumber?: boolean;
+  /**
+   * A map of components used to create the layout.
+   */
+  components?: Partial<Components>;
+
+  /** Content to add to the `tfoot` element. */
+  footer?: React.ReactNode;
+  // #endregion
+
+  // #region modifiers props
+
+  /**
+   * Apply the `disabled` modifier to the matching days.
+   */
+  disabled?: Matcher | Matcher[];
+  /**
+   * Apply the `hidden` modifier to the matching days. Will hide them from the
+   * calendar.
+   */
+  hidden?: Matcher | Matcher[];
+  /**
+   * The today’s date. Default is the current date. This Date will get the
+   * `today` modifier to style the day.
+   */
+  today?: Date;
+  /**
+   * Add modifiers to the matching days.
+   */
+  modifiers?: CustomModifiers;
+  /**
+   * The prefix to add to the modifiers class names. Default is `rdp-day_`.
+   *
+   * Each day will get a `${modifierPrefix}${modifier}` class name when matching
+   * a modifier.
+   */
+  modifierPrefix?: string;
+  // #endregion
+
+  // #region selection props
+  /**
+   * Toggle the controlled selection mode.
+   *
+   * - `custom`: customize what is selected
+   * - `single`: control the selection of single days
+   * - `multiple`: control the selection of multiple days
+   * - `range`: control the selection of a range of days
+   */
+  mode?: 'custom' | 'single' | 'multiple' | 'range';
+
+  // #region localization props
+  /**
+   * The date-fns locale object to localize the user interface. Defaults to
+   * `en-US`.
+   */
+  locale?: Locale;
+
+  /**
+   * A map of labels creators used for the ARIA labels attributes.
+   */
+  labels?: Partial<Labels>;
+
+  /**
+   * The text direction of the calendar. Use `ltr` for left-to-right (default)
+   * or `rtl` for right-to-left.
+   */
+  dir?: string;
```

Comment:
Could this be:
```
dir?: 'ltr' | 'rtl'
```

Reply:
I'd keep the HTML default for the HTMLDivElement which is `string`.

Reply:
Ok fair